sct-commons/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- SPDX-FileCopyrightText: 2021 RTE FRANCE -->
+<!-- -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.lfenergy.compas</groupId>
+        <artifactId>compas-sct</artifactId>
+        <version>local-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.lfenergy.compas</groupId>
+    <artifactId>sct-commons</artifactId>
+    <version>local-SNAPSHOT</version>
+
+    <properties>
+        <sonar.coverage.jacoco.xmlReportPaths>${basedir}/${aggregate.report.dir}</sonar.coverage.jacoco.xmlReportPaths>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
```

Comment:
Dependency not being used anymore?

Reply:
Do you guys have a white liste of libraries that RTE should use or not use.

There's also the following package in the POM, that are not used in this pull request. It will be certainly used in the next:
 
<dependency>
            <groupId>org.projectlombok</groupId>
            <artifactId>lombok</artifactId>
            <version>1.18.16</version>
            <optional>true</optional>
        </dependency>
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter-api</artifactId>
            <version>5.7.0</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.hamcrest</groupId>
            <artifactId>hamcrest</artifactId>
            <version>2.2</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.mockito</groupId>
            <artifactId>mockito-core</artifactId>
            <version>3.6.28</version>
            <scope>test</scope>
        </dependency>
        
Even the only class defined is useless at this moment. 

What's the criteria for a pull request.
Do we need weeks of discussion because a non releasable pull request contains empty conf file ?

Reply:
It's useless to add dependencies which are not being used.
You can better delete them :)
If they will be used in the next pull request, you can add them at that point.

By leaving dependencies in, you can forget about them and they will stay in even if they are not being used.

My advice would be: Delete them now, add them later if needed.

Reply:
Same with the config files by the way!
I see 2 empty files are added. They are useless right now, delete them.

If it's needed in the future, you can add them at that time.

Right now, it's just a waste.

Reply:
I bear wasting my time ... that's not a big deal for me at this moment

THe aim is to be sure all imports work with the build script (internal dependencies, external dependencies etc ... ). I guess that's understandable.

Reply:
Allright, it will be fixed in the future then! :)